### ISSUE TITLE
Using AWS root account credentials crashes scar

### ISSUE THREAD

**Issue by bcattle** [grycap/scar#180]
Using AWS [root account credentials](http://docs.aws.amazon.com/IAM/latest/UserGuide/id_root-user.html) crashes scar. 

```
$ scar init -n lambda-docker-cowsay -m 128 -t 300 grycap/cowsay
Traceback (most recent call last):
  File "/Users/bryan/Documents/scar/scar.py", line 931, in <module>
    CmdParser().execute()
  File "/Users/bryan/Documents/scar/scar.py", line 925, in execute
    args.func(args)
  File "/Users/bryan/Documents/scar/scar.py", line 82, in init
    Config.lambda_tags['owner'] = aws_client.get_user_name()
  File "/Users/bryan/Documents/scar/scar.py", line 564, in get_user_name
    return self.get_iam().get_user()['User']['UserName']
```

The `User` dict returned by root account contains a `UserId` field, but no `UserName`. 

```
(Pdb) self.get_iam().get_user()['User']
{'UserId': '712755113868', 'Arn': 'arn:aws:iam::712755113868:root', 'CreateDate': datetime.datetime(2017, 3, 29, 23, 17, 45, tzinfo=tzutc()), 'PasswordLastUsed': datetime.datetime(2017, 8, 1, 19, 16, tzinfo=tzutc())}
```

This value is being passed to `CreateFunction` via the `Tags` field. Hence it is optional and could be removed or replaced with the `UserId` field.

**Comment by gmolto** [comment]
Thank you @bcattle for the feedback. For sure you already know that using root credentials is highly discouraged when interacting with AWS (see http://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#lock-away-credentials). Instead, the best practice is to use an IAM user with the required privileges. Still your PR will be reviewed. Regards.

**Comment by alpegon** [comment]
Fixed in PR #181
